documents/meetup-starter-kit.md

```diff
@@ -0,0 +1,82 @@
+## Structure/Format
+
+Two to three presenters has been found to be best by multiple meetup coordinators.
+* 20-30 minute talks for each presenter is the sweet spot.
+  * Presentations run the risk of becoming too lengthy, and may lose interest of the audience.
+* You can specifically structure your event to a set number of speakers
+  * two speakers might be beneficial to give both a basic and an advanced talk, catering to the
+two groups.
+* Over your first few events, try different formats (two 30 minutes, three 20 minutes, two 20 minutes and 20 minutes of Q&A, and so on) and see which ones you think work best.
+
+As you start, solicit developers to come present at your meetup. You should
+plan out speakers a few events out. You want to be sure that you have people
+to speak at your event, and to be able to ensure speakers will have enough
+notice to ensure you're on their schedules.
+
+#### Protips:
+* "We generally have one beginner type talk and a more advanced one. Structuring our events this way allows us to reach a wide audience and have something more in depth for those that have been in the ecosystem for a while. Structuring our events this way allows us to reach a wide audience and have something more in depth for those that have been in the ecosystem for a while." @paulgrock on [GitHub](https://github.com/nodejs/evangelism/issues/162#issuecomment-145303436)
+* "Co-working spaces have been our go-to meetup venue hosts, but with attendance sometimes surpassing 100 people, it's a challenge to find a reliable, free venue." @danmactough on [GitHub](https://github.com/nodejs/evangelism/issues/162#issuecomment-145729309)
+* "We try to put on themed nights such as IoT, performance, testing but this takes a LOT of work to approach speakers and maybe a bit beyond beginner." @iancrowther on [GitHub](https://github.com/nodejs/evangelism/issues/162#issuecomment-145829320)
+* "We also run a segment called Community Announcements which is great as its very interactive with the audience. [...] Getting the audience on the mic makes networking super easy and ensures people know who to speak to and don't miss out. Everyone has had a few beers by then so its a lot of fun, some heckling and general mayhem. As a host, this is my favorite part of the event as I honestly have no ideas what's coming." @iancrowther on [GitHub](https://github.com/nodejs/evangelism/issues/162#issuecomment-145829320)
+
+
+## RSVP
+Using an online event/meetup planning site is the standard. [Meetup.com](http://www.meetup.com/) and
```

Comment:
Free RSVP can have up to a 90% no-show rate in major cities. Charging a small fee greatly increases he retention rate but if you stick with a free RSVP you may want to cap it higher than your actual capacity.